src/tabs/Pixnflix/index.tsx

```diff
@@ -134,6 +140,34 @@ class PixNFlix extends React.Component<Props, State> {
     }
   };
 
+  public onClickStill = () => {
+    const { mode } = this.state;
+    if (mode === 'still') {
+      this.handleSnapPicture();
+    } else if (mode === 'video') {
+      this.setState(
+        (state: State) => ({
+          ...state,

```

Comment:
Properties not specified when setting state do not get deleted, so it may not be necessary to destructure the existing state into the returned object (here and L163)